### PR TITLE
Improve AWS Setup Guide

### PR DESCRIPTION
**Pull request by sihu**

### CODE REVIEW

File: ruby_on_rails/aws.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+You will need Renuo-CLI to be set up and at the newest version:
+
+`gem install renuo-cli` --> see [renuo-cli](https://github.com/renuo/renuo-cli)
+
+Make sure `renuo -v` shows the [newest version](https://github.com/renuo/renuo-cli/tags)
+
+### Start the Setup
+
+The following command will guide you through a setup to be able to run all the commands, you
```
Comment (coorasse) [inline ruby_on_rails/aws.md]: can you rephrase? I cannot understand the meaning of this sentence

Comment (sihu) [reply ruby_on_rails/aws.md]: what about 
```md
The following command will create commands to set up S3 and CloudFlare. Before that, the command makes sure, you have the aws-profile and the aws-cli set up, so you have the necessary rights to run the commands.
```

Comment (coorasse) [reply ruby_on_rails/aws.md]: better, now I got it :)
